gooddata-sdk/gooddata_sdk/catalog_model/__init__.py

```diff
@@ -0,0 +1,10 @@
+# (C) 2022 GoodData Corporation
+
+from gooddata_sdk.catalog_model.data_source_content_objects import CatalogDataSourceTable
+from gooddata_sdk.catalog_model.workspace_content_objects import (
+    CatalogAttribute,
+    CatalogDataset,
+    CatalogFact,
+    CatalogLabel,
+    CatalogMetric,
+)
```

Comment:
What is the benefit of these imports?

Reply:
completely refactored, inner imports removed